packages/perseus/src/widgets/expression/expression.test.tsx

```diff
@@ -613,3 +618,90 @@ describe("Expression Widget", function () {
         });
     });
 });
+
+describe("Keypad configuration", () => {
+    it("should handle basic button set", async () => {
+        // Arrange
+        const widgetOptions: PerseusExpressionWidgetOptions = {
+            answerForms: [],
+            buttonSets: ["basic"],
+            times: false,
+            functions: [],
+        };
+
+        const expected: KeypadConfiguration = {
+            keypadType: KeypadType.EXPRESSION,
+            times: false,
+            scientific: false,
+            extraKeys: ["PI"],
+        };
+
+        // Act
+        const result = keypadConfigurationForProps(widgetOptions);
+
+        // Assert
+        expect(result).toEqual(expected);
```

Comment:
This may be preference, but I think it'd be much easier to trace this test if the expected were just declared here inline. Otherwise my brain has to backtrack up the test to find where it was defined (and it's not needed before this line): 

```suggestion
        expect(result).toEqual({
             keypadType: KeypadType.EXPRESSION,
             times: false,
             scientific: false,
             extraKeys: ["PI"],
         });
```

Reply:
No action required, but I think it's almost the same for the `widgetOptions`.